go.mod

```diff
@@ -8,12 +8,14 @@ require (
 	github.com/google/go-cmp v0.5.5
 	go.uber.org/zap v1.16.0
 	golang.org/x/net v0.0.0-20210119194325-5f4716e94777
+	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
 	k8s.io/api v0.19.7
 	k8s.io/apimachinery v0.19.7
 	k8s.io/client-go v9.0.0+incompatible
+	k8s.io/code-generator v0.19.7
```

Comment:
why is this needed ?

Reply:
Because update-codegen.sh depends on it.